### DISCUSSION TITLE
CVSS 4.0 scoring

### DISCUSSION THREAD

**Discussion by samueleresca** [aquasecurity/trivy#4586] [Ideas]
### Description

Hi,

CVSS 4.0 has been [announced](https://www.first.org/cvss/v4-0/index.html). The official publication is planned for Q4/2023.

Is there any plan for supporting that in trivy scans?
Do you have any timeline for that?

Thanks in advance.
Samuele

### Target

Container Image

### Scanner

Vulnerability

**Comment by knqyf263** [comment]
Are there any security advisories providing CVSS 4.0?

**Comment by knqyf263** [comment]
Created https://github.com/aquasecurity/trivy/issues/7059
